app/components/Wallet/ReceiveModal/ReceiveModal.js

```diff
@@ -58,116 +80,99 @@ class ReceiveModal extends React.Component {
       return null
     }
 
-    let qrCode
+    let qrCode, message
     switch (qrCodeType) {
       case QRCODE_TYPE_ADDRESS:
         qrCode = address
+        message = intl.formatMessage({ ...messages.copy_address })
         break
       case QRCODE_TYPE_PUBKEY:
         qrCode = pubkey
+        message = intl.formatMessage({ ...messages.copy_pubkey })
+        break
+      case QRCODE_TYPE_LNDCONNECT:
+        qrCode = lndConnect
+        message = intl.formatMessage({ ...messages.copy_uri })
         break
     }
 
     return (
       <Modal onClose={closeReceiveModal}>
-        <Flex justifyContent="center" alignItems="center" css={{ height: '100%' }}>
-          <Flex
-            justifyContent="space-between"
-            alignItems="center"
-            width={1}
-            bg="secondaryColor"
-            p={4}
-          >
-            <Flex as="section" flexDirection="column" alignItems="center" width={1 / 3}>
-              <Heading.h3 textAlign="center">
-                {alias && alias.length ? alias : pubkey.substring(0, 10)}
-              </Heading.h3>
-              <Flex justifyContent="center" mb={3}>
+        <Flex
+          justifyContent="space-between"
+          alignItems="center"
+          width={1}
+          bg="secondaryColor"
+          p={4}
+        >
+          <Flex as="section" flexDirection="column" alignItems="center" width={1}>
+            <Heading.h1 textAlign="center">
+              {alias && alias.length ? alias : pubkey.substring(0, 10)}
```

Comment:
I've seen `pubkey.substring(0, 10)` in multiple places (for instance in Network.js). Should be more flexible and convenient if we extract this into an util. Like `getShortPubKey`